doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -82,13 +82,16 @@ Matter
   * Default support for nRF7002 revision B.
   * Specific QR code and onboarding information in the documentation for each :ref:`Matter sample <matter_samples>` and the :ref:`Matter weather station <matter_weather_station_app>`.
   * The Bluetooth LE advertising arbiter class that enables easier coexistence of application components that want to advertise their Bluetooth LE services.
+  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the nRF52 family.
```

Comment:
```suggestion
  * Support for erasing settings partition during DFU over Bluetooth LE SMP for the Nordic nRF52 Series' SoCs.
```
As per https://projecttools.nordicsemi.no/confluence/display/MAR/Product+names

Reply:
Did this change get reverted?

Reply:
Yes, that was unintentional problem during rebase. It's fixed now.